src/Bicep.LangServer/Snippets/Templates/res-app-insights-auto-scale-settings.bicep

```diff
@@ -0,0 +1,62 @@
+﻿// Application Insights Auto Scale Settings
+resource ${1:appInsightsAutoScaleSettings} 'Microsoft.Insights/autoscalesettings@2015-04-01' = {
+  name: ${2:'name'}
+  location: resourceGroup().location
+  tags: {
+    Application_Type: '${3|web,other|}'
+    'hidden-link:${resourceGroup().id}/providers/Microsoft.Web/serverfarms/${4:appServicePlan}': 'Resource'

```

Comment:
This should be something like:
```
'hidden-link:${appService.id}': 'Resource'
```

Reply:
Updated here:
https://github.com/Azure/bicep/commit/ea170a6cefb1cfd227d5ffb7f8242f63aa441032